en/docs/references/enterprise-login.md

```diff
@@ -0,0 +1,70 @@
+# Enterprise Login
+
+Choreo's enterprise login feature allows your users residing in an external IdP (Identity Provider) to login into Choreo seamlessly without changing their credentials.
```

Comment:
Since we use the word "feature", shall we have "Enterprise Login" in title caps?

Reply:
done